pkg/cmd/alpha/alpha.go

```diff
@@ -0,0 +1,141 @@
+package alpha
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	jenkinsv1 "github.com/jenkins-x/jx/pkg/apis/jenkins.io/v1"
+	"github.com/jenkins-x/jx/pkg/cmd/helper"
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/cmd/templates"
+	"github.com/jenkins-x/jx/pkg/extensions"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/jenkins-x/jx/pkg/util"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// Options are the options to execute the alpha command
+type Options struct {
```

Comment:
Personally I think we should get away from this options pattern, especially here where there is no need for it at all. 
I'd rather have a `PluginManager` of sorts with a `GetPluginSpecs(url string)` and `RunPlugin`.